src/translator/sentence_ranges.cpp

```diff
@@ -99,5 +99,42 @@ string_view AnnotatedText::asStringView(const ByteRange &byteRange) const {
   return string_view(data, size);
 }
 
+string_view AnnotatedText::pre(size_t sentenceIdx) const {
+  // Find start of filler-text before, there's a corner case when there's no
+  // sentence before.
+  const char *start;
```

Comment:
Very minor comment, it's good practise to assign a ptr to `nullptr` when you declare it, but you don't actually assign its value right away.

Reply:
Set back to nullptr.